### ISSUE TITLE
No level file

### ISSUE THREAD

**Issue by JukoiDev** [clear-code-projects/Zelda#8]
Please give us the Tiled map file

**Comment by PeterRichardsWA** [comment]
i agree. this is useless if you dont have the Tiled file to load in the project which references all of the graphics.  The tutorial just kind of assumes you have it loaded magically.  Otherwise this is great stuff.  Very frustrating to hit Tiled part and essentially hit a brick wall. Please provide the project files for Tiled.
